core/src/main/java/org/apache/calcite/rel/core/Snapshot.java

```diff
@@ -94,7 +94,8 @@ public RexNode getPeriod() {
 
   @Override public boolean isValid(Litmus litmus, @Nullable Context context) {
     RelDataType dataType = period.getType();
-    if (dataType.getSqlTypeName() != SqlTypeName.TIMESTAMP) {
+    if (dataType.getSqlTypeName() != SqlTypeName.TIMESTAMP
+        && dataType.getSqlTypeName() != SqlTypeName.TIMESTAMP_WITH_LOCAL_TIME_ZONE) {
```

Comment:
This is done as a workaround to fix the tests.

Reply:
I'm fine with this workaround, but can we add a comment here to this effect? I'm a bit afraid that we'll forget that we've ever made this change if we don't mark it in some way.

Reply:
Ya sure.